test/asn1_internal_test.c

```diff
@@ -40,14 +40,15 @@ static int test_tbl_standard()
         last_nid = tmp->nid;
     }
 
-    if (last_nid != 0) {
-        fprintf(stderr, "asn1 tbl_standard: Table order OK\n");
+    if (TEST_int_ne(last_nid, 0)) {
+        TEST_info("asn1 tbl_standard: Table order OK");
         return 1;
     }
 
+    TEST_error("asn1 tbl_standard: out of order");
```

Comment:
I would argue against this. Or more specifically *combination* of this line and TEST_int_ne. Because in this line is printed you end up with two error outputs, which makes you believe that there were *two* errors.